### ISSUE TITLE
Green colour used in comparing articles does not meet WCAG's Colour Contrast

### ISSUE THREAD

**Issue by hydegeoff** [mmiron-HC/WST-RSAMS#173]
URL: https://hcdev.openplus.ca/en/node/60168/revisions/view/591938/597590/visual_inline
User Agent: Chrome
Language: English

Issue Description: When comparing two versions, there is green highlighted text that visually indicates the differences between two versions.  The green colour used to identify the differences between the versions does not meet colour contrast requirement set by WCAG.   The light green rgb(204, 255, 204) on white rgb(255, 255, 255) background does not meet the WCAG non-text colour contrast. It is currently 1.1:1 when it be should be 3:1. Currently, the text colour is valid, but when choosing a darker green, you must ensure text colour 4.5:1. See below:

Steps to reproduce:

1. Log into the Dashboard (RSAMS Dev environment)
2. Action on the "Build"
3. Select "Revisions" tab
4. To compare two revisions, use the radio buttons from column A and B and press the compare button
5. Green highlighted text appears as shown in the screenshot below.

Expected results:

The green colour should have 3:1 colour contrast to the background and text colour to green should have 4.5:1 colour contrast 
![Colour Contrast does not meet WCAG](https://user-images.githubusercontent.com/88147013/186426902-44d4806b-e74d-416d-9ca5-b029d0ec695c.jpg)

**Comment by echouin** [comment]
@hydegeoff this can be closed as the highlighting in green does not occur:
![image](https://user-images.githubusercontent.com/88099454/236250703-28588788-2f25-47d5-ac12-546eb3a98900.png)

**Comment by mmiron-HC** [comment]
@echouin  we will retest all these in the fall.

**Comment by echouin** [comment]
@mmiron-HC okay, I was just making notes for reference later. I am currently looking into solving some of the low-hanging fruit as well.
